qutebrowser/config/configdata.yml

```diff
@@ -1156,6 +1156,16 @@ completion.cmd_history_max_items:
 
     0: no history / -1: unlimited
 
+completion.fprompt_history_max_items:
```

Comment:
From the POV of a user, it's totally unclear what "fprompt" is supposed to be. Also, this isn't really related to the completion (which is invoked via `:`) at all. I'd recommend renaming `prompt.filebrowser` to `prompt.filebrowser.enabled`, and then renaming this to `prompt.filebrowser.history_max_items` or so.